src/components/deploymentConfig/DeploymentTemplateView.tsx

```diff
@@ -0,0 +1,770 @@
+import React, { useEffect, useState } from 'react'
+import Tippy from '@tippyjs/react'
+import { NavLink } from 'react-router-dom'
+import ReactSelect, { components } from 'react-select'
+import { MODES, ROLLOUT_DEPLOYMENT, URLS } from '../../config'
+import {
+    Checkbox,
+    CHECKBOX_VALUE,
+    ConditionalWrap,
+    isVersionLessThanOrEqualToTarget,
+    Progressing,
+    showError,
+    sortObjectArrayAlphabetically,
+    versionComparator,
+} from '../common'
+import { DropdownIndicator, Option } from '../v2/common/ReactSelect.utils'
+import { ReactComponent as Upload } from '../../assets/icons/ic-arrow-line-up.svg'
+import { ReactComponent as Arrows } from '../../assets/icons/ic-arrows-left-right.svg'
+import { ReactComponent as File } from '../../assets/icons/ic-file-text.svg'
+import { ReactComponent as Close } from '../../assets/icons/ic-close.svg'
+import { ReactComponent as Edit } from '../../assets/icons/ic-pencil.svg'
+import { ReactComponent as Next } from '../../assets/icons/ic-arrow-right.svg'
+import { MarkDown } from '../charts/discoverChartDetail/DiscoverChartDetails'
+import CodeEditor from '../CodeEditor/CodeEditor'
+import { getDeploymentTemplate } from './service'
+import { getDeploymentTemplate as getEnvDeploymentTemplate } from '../EnvironmentOverride/service'
+import YAML from 'yaml'
+import {
+    ChartTypeVersionOptionsProps,
+    CompareOptionsProps,
+    CompareWithDropdownProps,
+    DeploymentChartGroupOptionType,
+    DeploymentChartOptionType,
+    DeploymentConfigFormCTAProps,
+    DeploymentTemplateEditorViewProps,
+    DeploymentTemplateOptionsTabProps,
+} from './types'
+import { getCommonSelectStyles } from './constants'
+import { OrderBy } from '../app/types'
```

Comment:
Can we rename this to a more specific way

Reply:
Done. Changed to `SortingOrder`